portfolio/src/main/webapp/style.css

```diff
@@ -3,6 +3,52 @@
   margin-right: auto;
   width: 650px;
 }
+body {
+  background-image: url("images/Greywatercolor.jpg");
+  background-repeat: no-repeat;
+  background-attachment: fixed;
+  background-size: cover;
+}
+h1{
+    text-align: center;
+    font-family: 'Prata', serif ;
+}
+.nav { /*entire nav bar*/
+  text-align: center;
+}
+
+.nav li { /* item in in nav*/
+  display: inline-block;
+  
+}
+.nav li a { /*link in nav*/
```

Comment:
Great use of comments here to distinguish the different levels of granularity between `.nav`, `.nav li`, and `.nav li a`.